.github/workflows/sql-test-and-build-workflow.yml

```diff
@@ -48,7 +57,7 @@ jobs:
 
     # This step uses the codecov-action Github action: https://github.com/codecov/codecov-action
     - name: Upload SQL Coverage Report
-      if: always()
+      if: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
`if: always()` should be recovered, please, see my comment below

Reply:
That works, updated the PR